docs/dao-setup.md

```diff
@@ -57,6 +61,7 @@ rpcpassword=YOUR_PW
 blocknotify="%AppData%\Bitcoin\blocknotify.bat" %s
 ```
 
+
```

Comment:
Just curious if this added blank line was intentional/necessary?

Reply:
I'll remove this if all other resolved. (while editing in github i didnt see this line)